workspaces/jupyterlab-python/jupyter_server_config.py

```diff
@@ -19,5 +19,13 @@
 enable_rtc = bool(os.environ.get("ENABLE_REAL_TIME_COLLABORATION", False))
 c.YDocExtension.disable_rtc = not enable_rtc
 
+# Environmental variable to set the file that opens at launch of workspace
+# File is assumed to be in the workspace folder of the relevant question
+# RTC: needs to be appended if using the RTC Drive
+drive = "RTC:" if enable_rtc else ""
+
+if "LAUNCH_FILE_NAME" in os.environ:
+    c.LabApp.default_url= f"/lab/tree/{drive}{os.environ['LAUNCH_FILE_NAME']}"
```

Comment:
I usually like leaving breadcrumb links to GitHub issues, documentation, etc. when something isn't totally obvious:

```suggestion
# `LAUNCH_FILE_NAME` can be set to open a specific file when the workspace is opened.
if "LAUNCH_FILE_NAME" in os.environ:
    # The "RTC" drive needs to be used if real-time collaboration is enabled.
    # This isn't documented, but is discussed in these issues:
    # https://github.com/jupyterlab/jupyter-collaboration/issues/202
    # https://github.com/jupyterlab/jupyter-collaboration/issues/183
    drive = "RTC:" if enable_rtc else ""
    c.LabApp.default_url= f"/lab/tree/{drive}{os.environ['LAUNCH_FILE_NAME']}"
```

Reply:
Good point